src/joyride/nrepl.cljs

```diff
@@ -286,21 +286,24 @@
       (info "The nREPL server is already running")
       (p/rejected (js/Error. "The nREPL server is already running" {})))))
 
-(defn stop-server []
-  (debug "nREPL stop-server")
-  (if (server-running?)
-    (let [server (::server @!db)]
-      (.close server
-              (fn []
-                (swap! !db dissoc ::server)
-                (p/do
-                  (when-contexts/set-context! ::when-contexts/joyride.isNReplServerRunning false))
-                (-> (remove-port-file (::root-path @!db))
-                    (p/then
-                     (fn []
-                       (swap! !db dissoc ::root-path)
-                       (info "nREPL server stopped")))))))
-    (info "There is no nREPL Server running")))
+(defn stop-server+ []
+  (sci/alter-var-root sci/print-fn (constantly *print-fn*))
```

Comment:
Can you execute this after the promise is resolved?

Reply:
I think I can. Figured it was best to do it first thing. Why do you want us to wait?

Reply:
I figured there might still be messages being processed? Maybe it doesn't matter

Reply:
That's a good reason. =) I changed it.